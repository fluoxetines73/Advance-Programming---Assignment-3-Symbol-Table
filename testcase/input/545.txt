RPRINT
INSERT c string
PRINT
PRINT
ASSIGN c 'HelloWorld'
PRINT
BEGIN
ASSIGN c 'UPPERlower'
ASSIGN c 'UPPERlower'
END
PRINT
INSERT z number
PRINT
BEGIN
INSERT g_1 string
PRINT
LOOKUP g_1
END