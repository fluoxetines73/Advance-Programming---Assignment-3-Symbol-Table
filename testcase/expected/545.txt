
success
c//0
c//0
success
c//0
success
success
c//0
success
c//0 z//0
success
c//0 z//0 g_1//1
1